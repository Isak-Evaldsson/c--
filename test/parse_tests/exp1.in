# Compute the x'th fibonacci number.
6 + 31