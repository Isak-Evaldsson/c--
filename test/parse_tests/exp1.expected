BinaryExpr: +
    ConstExpr: 6
    ConstExpr: 31
